### PR TITLE
Added New Files

### PR DESCRIPTION
**Pull request by kan1shq**

### CODE REVIEW

**Comment by twitu** [comment]
You have created an extra directory. You can remove that later.

File: Kanishk Sharma/Kanishk Sharma/prob7-flatworm.hs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+-- flatworm (List [Elem 1, List [Elem 2, List [Elem 3, Elem 4], Elem 5]])
+
+flatworm :: NestedList a -> [a]
+
+flatworm (Elem x) = [x]
+
+flatworm (List x) = concatMap flatworm x   --was having a bit problem..so took help from a guy and got suggested 'concatMap'
```
Comment (twitu) [inline Kanishk Sharma/Kanishk Sharma/prob7-flatworm.hs:11]: As long as you can explain concatMap in the next lecture.

**Comment by twitu** [comment]
I'll merge it for now.

**Comment by kan1shq** [comment]
Yeah ok then, no problem.
Thanks tho, got to learn abt git thanks to u strictness😅.

**Pull request merged**